### PR TITLE
Batch of Updates

### PR DESCRIPTION
**Pull request by mosu-forge**
Updated Emission: I changed the 8.8M coins emitted on height 0 to 100k coins to correctly account for the burned premine. **Note: please let me know before you pull this to live** because I have an API at https://ryo-currency.com/supply/ which pulls from the block explorer and subtracts 8.7M outside of the explorer. This API needs to be updated to not subtract the 8.7M coins at the same time this commit is pulled live. **Note 2:** You must stop the explorer, delete `~/.ryo/lmdb02/emission_amount.txt`, and restart for this to take effect.

Updated templates: I updated tons of small things to improve mobile responsiveness. I also modifies the C++ side to add human readable number formats with thousands separator. I added a nice network stats block on the homepage.

Added API page: a simple static page outlining the API. Converted markdown to html and included code formatting CSS. 

Updated README: changed monero -> ryo and updated specific build instructions. Updated all API examples to Ryo domain and outputs.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -584,6 +584,11 @@ main(int ac, const char* av[])
 
         cout << "Enable JSON API\n";
 
+        CROW_ROUTE(app, "/api")
```
Comment (mosu-forge) [inline main.cpp:587]: This routes `/api` to the static page with API help.

File: src/CurrentBlockchainStatus.cpp [hunk 2]
```diff
@@ -137,6 +137,11 @@ CurrentBlockchainStatus::calculate_emission_in_blocks(
 
         uint64_t coinbase_amount = get_outs_money_amount(blk.miner_tx);
 
+        if(start_blk == 0 && coinbase_amount == 8800000000000000)
```
Comment (mosu-forge) [inline src/CurrentBlockchainStatus.cpp:140]: It's not necessary to have the `&&` clause, as either of these conditions is sufficient, but I used both for verbosity.

File: src/page.h [hunk 3]
```diff
@@ -486,6 +488,9 @@ class page
     index2(uint64_t page_no = 0, bool refresh_page = false)
     {
 
+        // Use commas for thousands separator
+        setlocale(LC_NUMERIC, "en_US.UTF-8");
```
Comment (mosu-forge) [inline src/page.h:492]: As far as I know, this doesn't negatively affect anything. I have not seen any issues with it on my test server.

File: src/page.h [hunk 4]
```diff
@@ -876,11 +886,15 @@ class page
             string emission_blk_no   = std::to_string(current_values.blk_no - 1);
             string emission_coinbase = xmr_amount_to_str(current_values.coinbase, "{:0.3f}");
             string emission_fee      = xmr_amount_to_str(current_values.fee, "{:0.3f}");
+            string emission_coinbase_human = fmt::format("{:n}", static_cast<int64_t>(current_values.coinbase/1e9));
+            string emission_fee_human = fmt::format("{:n}", static_cast<int64_t>(current_values.fee/1e9));
```
Comment (mosu-forge) [inline src/page.h:890]: libfmt is supposed to be able to format a float/double with thousand separator and fixed decimal places, but I get a run time error. So, I am casting to an int here which means circulating supply on front end will be a whole number. The API endpoint is not affected by this and returns the correct fractional amount.

File: src/CurrentBlockchainStatus.cpp (hunk 2 above)
Comment (fireice-uk) [reply src/CurrentBlockchainStatus.cpp:140]: Can you subtract `8700051446427001` instead? We want to be exact.

**Review by fireice-uk — CHANGES_REQUESTED** [review]

File: src/CurrentBlockchainStatus.cpp (hunk 2 above)
Comment (mosu-forge) [reply src/CurrentBlockchainStatus.cpp:140]: This has been fixed by changing `coinbase_amount` to `99948553572999` on height 0.

**Review by fireice-uk — APPROVED** [review]

**Pull request merged**